tests/test_card_schema_issue_111.py

```
@@ -0,0 +1,133 @@
+    "hub": SPARSE_REWARD,
+    "pretty": "Sparse Reward Long Tasks",
+    "blurb": "Sparse-reward leftover-goal long tasks (final reward only).",
+    "tags": ["synthetic-data", "sparse-reward", "long-horizon"],
+}
+SPARSE_SUMMARY = dict(
+    records=6551,
+    bytes_=67907183,
+    first="r01",
+    last="r6551",
+    names=["batch-r01.jsonl", "batch-r6551.jsonl"],
+)
```

Ruff (0.16.3)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)